src/main/resources/application.yml

```diff
@@ -11,7 +11,14 @@ spring:
     open-in-view: false
 bookstore:
   openapi:
-    dev-url: http://localhost:8080
+    dev-url: http://localhost:8080/api
 jwt:
   expiration: 1200000
-  secret: fkaafa1r88)_402492fls;l2=040;fs'104jfjlf1804180
\ No newline at end of file
+  secret: fkaafa1r88)_402492fls;l2=040;fs'104jfjlf1804180
+server:
+  servlet:
+    context-path: /api
```

Comment:
so you've removed the `/api` prefix from all the endpoint in favor of this property

what are the benefits? :) 

this adds up additional complexity, since from the reader prospective it is unclear, that all endpoints contain this prefix

also it is more error-prone - for example when implementing tests we may forgot about this property and spend some time wondering why `/books/{id}` doesn't work until we recollect that our context path is `/api` and the correct url would be `/api/books/{id}`

Reply:
Yea, I heard from someone that you can do this and not write the additional `/api` everywhere, but I myself ran into problems that it turned out to be not very convenient. So I'll return it the way it was.